xyo-android-library/src/main/java/network/xyo/sdk/XyoBleClient.kt

```diff
@@ -93,12 +93,14 @@ class XyoBleClient(
                                 return
                             }
                             (client as? XyoBridgeX)?.let {
+                                log.info("BridgeX rssi: $rssi");
```

Comment:
We will most likely remove these logs if this PR gets merged.

Reply:
Ok. No problem.